app/models/todo.rb

```diff
@@ -0,0 +1,2 @@
+class Todo < ApplicationRecord
```

Comment:
I think you missed the belongs_to helper in this model, right? otherwise, you won't be able to access the user this todo belongs to, as `@todo.user`